content/en/docs/concepts/windows/intro.md

```diff
@@ -0,0 +1,384 @@
+    * `spec.securityContext.fsGroup`
+    * `spec.securityContext.fsGroupChangePolicy`
+    * `spec.securityContext.sysctls`
+    * `spec.shareProcessNamespace`
+    * `spec.securityContext.runAsUser`
+    * `spec.securityContext.runAsGroup`
+    * `spec.securityContext.supplementalGroups`
+    * `spec.containers[*].securityContext.seLinuxOptions`
+    * `spec.containers[*].securityContext.seccompProfile`
+    * `spec.containers[*].securityContext.capabilities`
+    * `spec.containers[*].securityContext.readOnlyRootFilesystem`
+    * `spec.containers[*].securityContext.privileged`
+    * `spec.containers[*].securityContext.allowPrivilegeEscalation`
+    * `spec.containers[*].securityContext.procMount`
+    * `spec.containers[*].securityContext.runAsUser`
+    * `spec.containers[*].securityContext.runAsGroup`
+
+    In the above list, wildcards (`*`) indicate all elements in a list.
+    For example, `spec.containers[*].securityContext` refers to the SecurityContext object
+    for all containers. If any of these fields is specified, the Pod will
+    not be admited by the API server.
+
+* [Workload resources](/docs/concepts/workloads/controllers/) including:
+  * ReplicaSet
+  * Deployments
+  * StatefulSets
+  * DaemonSet
+  * Job
+  * CronJob
+  * ReplicationController
+* {{< glossary_tooltip text="Services" term_id="service" >}}
+  See [Load balancing and Services](#load-balancing-and-services) for more details.
+
+Pods, workload resources, and Services are critical elements to managing Windows
+workloads on Kubernetes. However, on their own they are not enough to enable
+the proper lifecycle management of Windows workloads in a dynamic cloud native
+environment. Kubernetes also supports:
+
+* `kubectl exec`
+* Pod and container metrics
+* {{< glossary_tooltip text="Horizontal pod autoscaling" term_id="horizontal-pod-autoscaler" >}}
+* {{< glossary_tooltip text="Resource quotas" term_id="resource-quota" >}}
+* Scheduler preemption
+
+### Command line options for the kubelet {#kubelet-compatibility}
+
+The behavior of some kubelet command line options behave differently on Windows, as described below:
+
+* The `--windows-priorityclass` lets you set the scheduling priority of the kubelet process (see [CPU resource management](/docs/concepts/configuration/windows-resource-management/#resource-management-cpu))
+* The `--kubelet-reserve`, `--system-reserve` , and `--eviction-hard` flags update [NodeAllocatable](/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable)
+* Eviction by using `--enforce-node-allocable` is not implemented
+* Eviction by using `--eviction-hard` and `--eviction-soft` are not implemented
+* A kubelet running on a Windows node does not have memory
+  restrictions. `--kubelet-reserve` and `--system-reserve` do not set limits on
+  kubelet or processes running on the host. This means kubelet or a process on the host
+  could cause memory resource starvation outside the node-allocatable and scheduler.
```

Comment:
“This means kubelet or a process on the host could cause memory resource starvation outside the node-allocatable and scheduler.”

This sentence is difficult to parse.